indico/modules/events/forms.py

```diff
@@ -68,3 +68,14 @@ def __init__(self, *args, **kwargs):
 
     def is_submitted(self):
         return super(EmailEventPersonsForm, self).is_submitted() and 'submitted' in request.form
+
+
+class GeneralSettingsReferenceTypeForm(IndicoForm):
```

Comment:
`EventReferencesForm` - it's not a reference type form, and `GeneralSettings` is way too frontend-specific